ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__multi-nightly.yaml

```diff
@@ -571,6 +571,34 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-azure-ipi-fullyprivate-proxy
+- as: azure-ipi-fullyprivate-internal-registry-amd-f28-destructive
+  cron: 44 14 20 * *
```

Comment:
psl run `ci-operator/config/openshift/openshift-tests-private/tools/generate-cron-entry.sh <job-name> <ci-yaml_file_name>` to generate an uniq cron.